grails-app/controllers/kuorum/users/ProfileController.groovy

```diff
@@ -545,6 +535,11 @@ class ProfileController {
             command.errors.rejectValue("phone", "kuorum.web.commands.profile.funnel.FunnelFillBasicDataCommand.phone.repeated")
             handleSaveFunnelFillBasicDataErrorCommand(command, user)
             return;
+        } finally{
+            Map<String,String> extraInfoList = new HashMap<>();
+            extraInfoList.put(command.contactName,command.phone);
+            KuorumUserSession userSession = springSecurityService.principal
+            contactService.putExtraInfo(userSession, contact.getId(), extraInfoList)
```

Comment:
Remove Finally
And recover ExtraInfoList, if its empty create one, if not, reuse the extraInfo